XCUITests/ScreenGraph.swift

```diff
@@ -165,14 +165,18 @@ private let hittablePredicate = NSPredicate(format: "hittable == true")
 private let noopNodeVisitor: NodeVisitor = { _ in }
 
 extension ScreenGraphNode {
-    fileprivate func waitForElement(_ element: XCUIElement, withTest xcTest: XCTestCase, handler: @escaping XCWaitCompletionHandler) {
+    fileprivate func waitForElement(_ element: XCUIElement, withTest xcTest: XCTestCase, notFoundHandler: () -> ()) {
+        // Quick check to avoid waiting.
         if element.exists {
```

Comment:
I don't think XCTWaiter().wait (or waitForExpectations) polls - in my experience it exits immediately when the expectation is satisfied, so we won't lose time if we remove this if clause.  what do you think?

Reply:
:+1: yes I think you're right. 

Superstition made me leave this in; but it's Apple's bug if they're break it. :)